test_unit.py

```diff
@@ -963,7 +965,7 @@ def test_host_validation(
     ):
         host_input = {"ansible_host": "some ansible host", "account": "some acct"}
 
-        deserialize_host(host_input)
+        deserialize_host(host_input, host_schema)
```

Comment:
```suggestion
        from unittest.mock import MagicMock
        host_schema = MagicMock()
        deserialize_host(host_input, host_schema)
```

_MagicMock_ instead of just _Mock_, because it supports magic methods like _\_\_getitem\_\__. Item access is used inside of the deserialization.